android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java

```diff
@@ -84,22 +100,43 @@ public boolean onTouch(View v, MotionEvent event) {
         });
 
         mSpinner = view.findViewById(R.id.spinner);
-        mSpinner.setOnItemSelectedListener(this);
+        // class PortfolioFragment in fact is used both for Portfolio and Apps
+        // screens, see CryptoFragmentPageAdapter.getItem.
+        // Without post task with delay it happens that when 2nd instance is
+        // created, then 1st instance onItemSelected is triggered with
+        // position=0. That makes onItemSelected to switch network, but the
+        // actual displayed value of spinner's item on 1st instance remains
+        // unchanged.
+        PostTask.postDelayedTask(UiThreadTaskTraits.DEFAULT,
+                () -> { mSpinner.setOnItemSelectedListener(this); }, 500);
+
+        mBalance = view.findViewById(R.id.balance);
+        mBalance.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                UpdatePortfolio();
+            }
+        });
 
         // Creating adapter for spinner
         ArrayAdapter<String> dataAdapter = new ArrayAdapter<String>(getActivity(),
                 android.R.layout.simple_spinner_item, Utils.getNetworksList(getActivity()));
         dataAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
         mSpinner.setAdapter(dataAdapter);
 
+        RadioGroup radioGroup = view.findViewById(R.id.portfolio_duration_radio_group);
```

Comment:
unused var?

Reply:
yes, indeed

Reply:
Fixed in https://github.com/brave/brave-core/pull/10174/commits/4c0d233d43308b0dfd339f73d69ac9fff4b4d458